src/functions_framework/__init__.py

```diff
@@ -180,18 +240,11 @@ def create_app(target=None, source=None, signature_type=None):
         app.view_functions["run"] = _http_view_func_wrapper(function, flask.request)
         app.view_functions["error"] = lambda: flask.abort(404, description="Not Found")
     elif signature_type == "event":
-        app.url_map.add(
-            werkzeug.routing.Rule(
-                "/", defaults={"path": ""}, endpoint="run", methods=["POST"]
-            )
-        )
-        app.url_map.add(
-            werkzeug.routing.Rule("/<path:path>", endpoint="run", methods=["POST"])
-        )
+        _setup_event_routes(app)
         app.view_functions["run"] = _event_view_func_wrapper(function, flask.request)
-        # Add a dummy endpoint for GET /
-        app.url_map.add(werkzeug.routing.Rule("/", endpoint="get", methods=["GET"]))
-        app.view_functions["get"] = lambda: ""
+    elif signature_type == "cloudevent":
+        _setup_event_routes(app)
+        app.view_functions["run"] = _cloudevent_view_func_wrapper(function, flask.request)
```

Comment:
This is a nit but I think I'm OK with this being duplicated twice inline rather than factored out. As-is this makes it kind of hard to follow what routes are being created for what signatures.

Given how similar they are now, we could even do:

```python
elif signature_type == "event" or signature_type == "cloudevent":
    app.url_map.add(
        werkzeug.routing.Rule(
            "/", defaults={"path": ""}, endpoint=signature_type, methods=["POST"]
        )
    )
    app.url_map.add(
        werkzeug.routing.Rule("/<path:path>", endpoint=signature_type, methods=["POST"])
    )

    # Add a dummy endpoint for GET /
    app.url_map.add(werkzeug.routing.Rule("/", endpoint="get", methods=["GET"]))
    app.view_functions["get"] = lambda: ""
    
    # Add the view functions
    app.view_functions["cloudevent"] = _event_view_func_wrapper(function, flask.request)
    app.view_functions["cloudevent"] = _cloudevent_view_func_wrapper(function, flask.request)
```

Doesn't really hurt to add a view function that will never be used.